backend/utils/dto.py

```diff
@@ -0,0 +1,61 @@
+import json
+from datetime import datetime
+from typing import Any, Optional
+
+from unstract.workflow_execution.enums import LogType
+
+from unstract.core.constants import LogFieldName
+
+
+class LogDataDTO:
+    """Log data DTO
+    DTO for log data to store to QUEUE
+    Attributes:
+        execution_id: execution id
+        organization_id: organization id
+        timestamp: timestamp
+        log_type: log type
+        data: log data
+    """
+
+    def __init__(
+        self,
+        execution_id: str,
+        organization_id: str,
+        timestamp: int,
+        log_type: str,
+        data: dict[str, Any],
+    ):
+        self.execution_id: str = execution_id
+        self.organization_id: str = organization_id
+        self.timestamp: int = timestamp
+        self.event_time: datetime = datetime.fromtimestamp(timestamp)
+        self.log_type: LogType = log_type
+        self.data: dict[str, Any] = data
+
+    @classmethod
+    def from_json(cls, json_data: str) -> Optional["LogDataDTO"]:
+        try:
+            json_data = json.loads(json_data)
+            execution_id = json_data.get(LogFieldName.EXECUTION_ID)
+            organization_id = json_data.get(LogFieldName.ORGANIZATION_ID)
+            timestamp = json_data.get(LogFieldName.TIMESTAMP)
+            log_type = json_data.get(LogFieldName.TYPE)
+            data = json_data.get(LogFieldName.DATA)
+            if all((execution_id, organization_id, timestamp, log_type, data)):
+                return cls(execution_id, organization_id, timestamp, log_type, data)
+        except (json.JSONDecodeError, AttributeError):
+            pass
```

Comment:
@muhammad-ali-e @athul-rs can we add debug log here, will be helpful to debug something happens here.

Reply:
addressed